### DISCUSSION TITLE
Teslafi Import after first Teslamate Datapoint

### DISCUSSION THREAD

**Discussion by jonchen727** [teslamate-org/teslamate#867] [Ideas]
**What would you like to be added**:
Allow teslamate to fill gaps in data from a teslafi/other logger import

**Why is this needed**:
Fill missing when teslamate was down from redundant sources 

**Additional context**:

**Comment by BabyYeggie** [comment]
For further information, this is due to the unique index used in the database. Attempts to import data before/between drives/charges/etc. Attempts to insert data will have the following error:

ERROR:  duplicate key value violates unique constraint "positions_pkey"
DETAIL:  Key (id)=(xxxx) already exists.
